viewer/src/main/webapp/viewer-html/components/ExtendedFeatureInfo.js

```diff
@@ -73,6 +73,7 @@ Ext.define ("viewer.components.ExtendedFeatureInfo",{
         this.getViewerController().mapComponent.getMap().addListener(viewer.viewercontroller.controller.Event.ON_GET_FEATURE_INFO,this.onFeatureInfoStart,this);
         //listen to a extent change
         this.getViewerController().mapComponent.getMap().addListener(viewer.viewercontroller.controller.Event.ON_CHANGE_EXTENT, this.onChangeExtent,this);
+        this.config.viewerController.addListener(viewer.viewercontroller.controller.Event.ON_EDIT_SUCCESS, this.featureEdited, this);
```

Comment:
odd to use `this.config.viewerController` here while bove we use `this.getViewerController()`